### PR TITLE
rocprofiler-dev update upto 6.0

### PR DESCRIPTION
**Pull request by renjithravindrankannath**
rocprofiler-dev update upto 6.0

### CODE REVIEW

File: var/spack/repos/builtin/packages/rocprofiler-dev/package.py [hunk 1]
```diff
@@ -126,6 +134,13 @@ class RocprofilerDev(CMakePackage):
         "5.3.3",
         "5.4.0",
         "5.4.3",
+        "5.5.0",
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/rocprofiler-dev/package.py:136]: Did you mean to add `"5.4.4"` here too?

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Confirmed all of the version sha256s.  I *think* you are missing one of the versions from the list.  Let me know if I'm mistaken.

File: var/spack/repos/builtin/packages/rocprofiler-dev/package.py (hunk 1 above)
Comment (renjithravindrankannath) [reply var/spack/repos/builtin/packages/rocprofiler-dev/package.py:136]: Removed 5.4.4 as there is no 5.4.4 releases for "hsakmt-roct and hsa-rocr-dev.

**Comment by tldahlgren** [comment]
Not sure what to make about the CI failure for the package.  It's at https://gitlab.spack.io/spack/spack/-/jobs/9849185.

**Review by tldahlgren — APPROVED** [review]

**Comment by renjithravindrankannath** [comment]
Closing this PR as #42459 will have the changes to handle rocprofiler-dev.

**Pull request closed without merging**